samples/CustomMiddleware/CustomMiddleware.csproj

```diff
@@ -13,9 +13,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.8.0-preview1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.9.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.5.0-preview1" OutputItemType="Analyzer" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Storage" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0" />
```

Comment:
Don't we want to keep `OutputItemType="Analyzer"`?

Reply:
Nope.

Reply:
Just for my understanding why? :D

Reply:
It is needed only when you add as a project reference. For package reference, that is not needed.

Reply:
Good to know, we should probably remove this from templates next time we do an update

Reply:
It is already removed from templates.